### PR TITLE
Add serviceStatus and serviceSendAction admin APIs

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: src/Network/Minio/AdminAPI.hs [hunk 1]
```diff
@@ -344,6 +385,35 @@ healPath bucket prefix = do
...
+serviceSendAction action = do
+    let payload = PayloadBS $ LBS.toStrict $ A.encode action
+    _ <- executeAdminRequest AdminReqInfo { ariMethod = HT.methodPost
+                                          , ariPayload = payload
+                                          , ariPayloadHash = Nothing
+                                          , ariPath = "v1/service"
+                                          , ariHeaders = []
+                                          , ariQueryParams = []
+                                          }
+    return $ ()
```
Comment (donatello) [inline src/Network/Minio/AdminAPI.hs]: The `$` is not needed above. 

Also, to ignore the returned value, you can do:

```haskell
void $ executeAdminRequest ...
```

Comment (harshavardhana) [reply src/Network/Minio/AdminAPI.hs]: Done

File: src/Network/Minio/AdminAPI.hs [hunk 2]
```diff
@@ -344,6 +385,35 @@ healPath bucket prefix = do
...
+serviceSendAction action = do
+    let payload = PayloadBS $ LBS.toStrict $ A.encode action
+    void $ executeAdminRequest AdminReqInfo { ariMethod = HT.methodPost
+                                            , ariPayload = payload
+                                            , ariPayloadHash = Nothing
+                                            , ariPath = "v1/service"
+                                            , ariHeaders = []
+                                            , ariQueryParams = []
+                                            }
+    return ()
```
Comment (donatello) [inline src/Network/Minio/AdminAPI.hs]: You can drop the return line because of the void.

Comment (harshavardhana) [reply src/Network/Minio/AdminAPI.hs]: done

**Pull request merged**